packages/create-app/templates/default-app/packages/backend/src/plugins/techdocs.ts

```diff
@@ -28,7 +28,7 @@ export default async function createPlugin({
   preparers.register('github', commonGitPreparer);
   preparers.register('gitlab', commonGitPreparer);
 
-  const publisher = new LocalPublish(logger, discovery);
+  const publisher = Publisher.fromConfig(config, logger, discovery);
```

Comment:
Could you document these changes to `create-app` in a separate changeset?

Reply:
@Rugvip Sure! I'll add it as a `patch` change for `@backstage/create-app` but I am not sure if it's `patch` or `minor`, since the changes in techdocs plugins are `minor`.

Reply:
Patch is fine, minor would be if we change the way you interact with the `create-app` CLI itself